scripts/dashboard/medperf_dashboard/assets/stages.csv

```diff
@@ -0,0 +1,51 @@
+4,Tumor Extract Finished,,,pink
+-4,Tumor Extract Failed,,,rebeccapurple
+5,Manual Review Completed,,,mediumslateblue
+-5,Manual Review Required,"Baseline tumor segmentations have been generated. Manual corrections are required to proceed. Here are the following recommended methods.
+
+### Local Manual Corrections
+If you're doing manual corrections on this machine, and you're running the manual Data Preparation pipeline locally,  you may find buttons at the bottom of this page to aid you in the process of manual correction.  These buttons automatically start the review process with ITK-Snap using the generated tumor segmentation mask.  Please ensure ITK-Snap has been installed with client-line tools to use this method. You may edit the file as many times as needed. Once you're done with this subject, press the ""Mark as finalized"" button to let the pipeline know the subject is done.
+
+In some cases, local manual correction is not possible, or the default baseline segmentation is not sufficiently good to work with.  In those cases, it is recommended to do manual correction through the **Packaging method**.
+
+### Packaging Method
+If using the monitoring tool, you may package cases for review by pressing the 'Package cases for review' button under **SUMMARY**. This will create a tarball file on the working directory (displayed on the monitoring tool header), containing all the cases for review. You may untar the segmentation with the following command
+
+``` tar -xzf review_cases.tar.gz ```
+
+If working remotely, move the tarball file to your local machine. Untar the tarball file, select the best baseline segmentation and make any necessary corrections.  Once you're done, move the finalized file to the `finalized` folder, retaining the original name.
+
+If using the monitoring tool, create a new tarball with the previously untarred files, as well as finalized cases. You can do so with the following command
+
+``` tar -czf reviewed_cases.tar.gz -C review_cases . ```
+
+Place this new tarball file on the directory the monitoring tool is running (displayed on the monitoring tool header). The tool will automatically detect the finalized cases and place them in the expected locations.
+
+### Brain Mask Correction
+There might be situations where the brain mask is not correct. If working locally you may use the `Review Brain Mask` button to automatically make brain mask corrections with ITK-Snap. If instead you're using the previously menitoned tarball, you may find the brain mask segmentation as `brainMask_fused.nii.gz`. Please make the necessary corrections under that file, overwriting the contents. Once the new tarball is created and placed within the directory the monitoring tool is running, it will be automatically identified and placed in the correct location.
+**NOTE** Modifying the brain mask invalidates the tumor segmentations, and re-runs the tumor segmentation procedure using this new mask.
+",,mediumseagreen

```

Comment:
is the text here intentional?

Reply:
It seems to me this is just a multiline str value of comment field for the `-5` stage. At least csv is parsed successfully

Reply:
And I don't see any usage of this `comment` field in the dashboard. I believe it is here just because `stages.csv` file was taken as it is used in monitor (?). Anyway, it does not break anything